components/InputArea/index.html

```diff
@@ -0,0 +1,90 @@
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='../../index.html'> Home</a>
+              <a class='li-link' href='components/alert/index.html'>Documentation</a>
+              <a class='li-link' target='_blank' href='https://github.com/vineeth-t/Main/tree/Devlopment'>Github</a>
+             
+        </div>
+        </nav>
+    <header class='header' >
+        <div class='menu-list' id='menu-items'>
+            <a class="li-link component" href='/components/Alert/index.html'>Alert</a>
+            <a class="li-link component" href='/components/Avatar/index.html'>Avatar</a>
+            <a class="li-link component" href='/components/Badge/index.html'>Badge</a>
+            <a class="li-link component" href='/components/Buttons/index.html'>Button</a>
+            <a class="li-link component" href='/components/Card/index.html'>Cards</a>
+            <a class="li-link component link-active " href='/components/InputArea/index.html'>Input</a>
+            <a class="li-link component" href='/components/List/index.html'>Lists</a>
+            <a class="li-link component" href='/components/Toast/index.html'>Toast</a>
+            <button class='Float-btn Floating-btn-x' onclick="floatingOff()">X</button>
+            
+        </div>
+        <div class='component-content'>
+            <button class='Float-btn Floating-btn' onclick="floatingAction()"><></button>
+            <div>
+                <h2>Input Tag</h2>
+                <p class='content-txt'>Input area helps to save the user data,thoughts,passwords to server </p>
+            </div>  <hr><br/>
+            <div>
+                <h2>Input with labels</h2>
+                <div class='content-txt'>
+                    <label>Username :</label>
+                    <input type='text'/>
+                </div>
+                <div class='content-txt'>
+                    <label>Password :</label>
+                    <input type='password'/>
+                </div>
+            </div><hr><br/>
+            <div>
+                <h2>Form Handling</h2>
+                <form onsubmit="" >
+                <div class='content-txt'>
+                    <label>Email-Id <span class='asteric-color'>*</span>&nbsp&nbsp&nbsp&nbsp: </label>
+                    <input type='text'/>
+                </div>
+                <div class='content-txt'>
+                    <label>Password <span class='asteric-color'>*</span>:</label>
+                    <input type='password'/>
+                </div>
+                <div class='content-txt'>
+                    <label>Mobile No<span class='asteric-color'>*</span> :</label>
+                    <input type='number'/>
+                </div>
+                <div class='content-txt'>
+                  <input type='submit' value='submit' disabled>
```

Comment:
Conventionally, buttons are used for submitting an action. 
Also Buttons can be provided the class you defined earlier i.e., `btn` to style them quickly.